cmd/geninstaller/install.sh.tmpl

```diff
@@ -7,6 +7,15 @@
 
 set -eo pipefail
 
+if [ -z "${USER}" ]; then
```

Comment:
Why don't we ditch $USER/$GID and use `id -u`/`id -g`?

Reply:
Ah, nice. Will see if that works with alpine